tools/cockpit.spec.in

```diff
@@ -125,10 +124,8 @@ BuildRequires: gdb
 # For documentation
 BuildRequires: xmlto
 
-%if 0%{?with_selinux}
 BuildRequires:  selinux-policy
 BuildRequires:  selinux-policy-devel
-%endif
```

Comment:
FTR, I think it's fine for cockpit-appstream to buildrequires: selinux-policy; it doesn't really hurt, and it simplifies the spec a bit.

However, this change does not match up with the commit message:

> without a static BuildRequires on selinux-policy, the macros themselves aren't available during the SRPM build

This *is* a static BuildRequires, so what does that mean?

> instead of using %selinux_requires

main doesn't use that macro either. What does it do? Could we use it now, because selinux-policy becomes a static buildreq?
Also, neither main nor this PR

Reply:
Right.  So I changed course here and ended up not using the `%selinux_requires` macro because it was too much of a pain in the butt.  I'll have to fix this message up.

Reply:
Er.  I rewrote the commit message already, though.  What's there is true.  This point form list is a list of reasons that we *can't* use the macro and need to use our open-coded versions instead.

Can you suggest some wording that would make that more clear?

Reply:
As per IRC: So this part documents the status quo, and how the `%selinux_requires` macro is conceptually broken (it requires its own package already anyway). It does not apply to the change here, which is what confused me (also, the bit about the "no static build requires", which we clearly have). I'd just drop the paragraph altogether, honestly.

Reply:
Would it help if I changed "without a static BuildRequires on selinux-policy, the macros themselves aren't available during the SRPM build" to "...*wouldn't be* available ..." ?

Reply:
That would help a bit indeed, yes; to point out that this "without a static dep" is a hypothesis, not a description of the status quo. Thanks!

Reply:
Done.